index.js

```diff
@@ -163,4 +178,30 @@ PagedContacts.authorized = PagedContactsModule.authorized;
 if (Platform.OS === 'ios') {
   PagedContacts.notDetermined = PagedContactsModule.notDetermined;
   PagedContacts.restricted = PagedContactsModule.restricted;
-}
\ No newline at end of file
+}
+
+/**
+ * The `PagedContactsImage` class is a simple wrapper around the `Image` component
+ * using `PagedContacts.setImageViewWithHandle`
+ * function to update the view with the contact picture on the native side.
+ * 
+ * @prop {Number} contactId Contact identifier
+ * @prop {String} imageType Image type - either `image` or `thumbnailImage`
+ * 
+ * @memberOf PagedContacts
+ */
+class PagedContactsImage extends Component {
```

Comment:
I don’t like this. Instead of a simple and intuitive contacts framework, we now have a behemoth. This does not solve the send over network issue; would we also write a network framework to send contacts?

Instead of reinventing the wheel, did we investigate why there are performance issues in the first place? Thumbnails, in particular, should not cause performance issues over the bridge. Perhaps Android returns huge “thumbnails” which we can resize in native Android. Do we have metrics to show us what is slow?

CC @talkol What do you think?